blocks/columns/columns.css

```diff
@@ -167,3 +169,77 @@
   max-width: 520px;
   max-height: 293px;
 }
+
+.columns.carousel button{
+  align-items: flex-start;
+  width: 10px;
+  height: 10px;
+  border-radius: 100%;
+  margin: 10px;
+  padding: 0;
+  background-color: var(--black-color);
+  border: none;
+}
+
+@media (min-width: 990px) {
+  .columns.carousel button{
+    margin: -5px;
+  }
+
+  .columns.carousel > div:nth-child(1), .columns.carousel > div:nth-child(2) {
+    left: 175px !important;
```

Comment:
Why so many uses of `!important` ? The cascading part should be rather clear to understand and be able to combine multiple styles together.